en/02_Developer_Guides/05_Extending/01_Extensions.md

```diff
@@ -44,63 +45,137 @@ Convention is for extension class names to end in `Extension`. This isn't a requ
 [/info]
 
 After this class has been created, it does not yet apply it to any object. We need to tell Silverstripe CMS what classes 
-we want to add the `MyMemberExtension` too. To activate this extension, add the following via the [Configuration API](../configuration).
+we want to add the `MyMemberExtension` to. To activate this extension, add the following via the [Configuration API](../configuration).
 
 **app/_config/app.yml**
 
-
 ```yml
 SilverStripe\Security\Member:
   extensions:
-    - MyMemberExtension
+    - App\Extensions\MyMemberExtension
 ```
 
 Alternatively, we can add extensions through PHP code (in the `_config.php` file).
 
-
 ```php
 SilverStripe\Security\Member::add_extension(MyMemberExtension::class);
 ```
 
 This class now defines a `MyMemberExtension` that applies to all `Member` instances on the website. It will have 
 transformed the original `Member` class in two ways:
 
-* Added a new [DBDatetime](api:SilverStripe\ORM\FieldType\DBDatetime) for the users date of birth, and;
-* Added a `SayHi` method to output `Hi <User>`
+* Added a new [DBDatetime](api:SilverStripe\ORM\FieldType\DBDatetime) database field for the user's date of birth, and;
+* Added a public `SayHi` method to output `Hi <User>`
 
 From within the extension we can add more functions, database fields, relations or other properties and have them added
 to the underlying `DataObject` just as if they were added to the original `Member` class but without the need to edit
 that file directly.
 
+## Owner
+
+In your [Extension](api:SilverStripe\Core\Extension) class you can only refer to the source object through the `owner` property on the class as 
+`$this` will refer to your `Extension` instance.
+
+```php
+use SilverStripe\ORM\DataExtension;
+
+class MyMemberExtension extends DataExtension 
+{
+    public function updateFoo($foo) 
+    {
+        // outputs the class name of the object being extended
+        var_dump(get_class($this->owner));
+    }
+}
+```
+
+## Adding configuration properties
+
+Extension classes can add to configuration properties for the classes they extend. Any configuration property declared in an extension can be accessed both from the context of the extension class itself as well as from the context of the class being extended.
+
+```php
+namespace App\Data;
 
-### Adding Database Fields
+use SilverStripe\View\ViewableData;
+
+class MyDataClass extends ViewableData
+{
+    private static array $my_configuration_property = [
```

Comment:
Probably makes sense to use `private static array $db` for these examples since that's going to be the most common config done in extensions

Reply:
This section is about configuration as an agnostic concept. DB fields are _explicitly_ covered in the next section. This is to make it clear that _any_ config can be added in extensions.